rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -0,0 +1,112 @@
+local json = require "json"
```

Comment:
Is this a plain Lua JSON lib? To my knowledge `cjson` is the recommended library to use for JSON encoding/decoding.

Reply:
As far as I know `cjson` cannot be used with LuaJIT.

Reply:
I have been using it with LuaJIT, also Openresty includes it by default https://openresty.org/en/lua-cjson-library.html